README.md

```diff
@@ -2,10 +2,20 @@
 
 This repository is a place to put config files related to code and programming style checks.
 
-Currently this only includes checkstyle configs (see [the checkstyle documentation](http://checkstyle.sourceforge.net/)), but it could conceivably also include configs for related tools like PMD and findbugs.
+Currently this supports both Java and Scala configurations, but it could conceivably also include configs for other languages and tools like PMD and findbugs.
 
-## Usage
+### Java ###
 
+This includes ide (both IntelliJ and Eclipse) and checkstyle configs for java (see [the checkstyle documentation](http://checkstyle.sourceforge.net/)).
+
+### Scala ###
+
+Currently this only includes scalastyle configs (see [the scalastyle documentation](http://www.scalastyle.org/))
```

Comment:
@zdavep suggested using scalariform to automatically fix style issues

Reply:
We can certainly look at using scalariform to fix style issues. I wasn't sure we wanted something to make changes to our code, following the java checkstyle pattern I was more focusing on warnings about issues than just changing things